mars/Pages/LanguagePage.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Mars.Support;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mars.Pages
+{
+    public class LanguagePage
+    {
+        public readonly By language_tab = By.XPath("//a[text()=\"Languages\"]");
+
+        public readonly By languange_question_text = By.XPath("//div[contains(text(), 'How many languages do you speak?')]");
+
+        public readonly By add_language_input = By.XPath("//input[@placeholder=\"Add Language\"]");
+        public readonly By language_level_dropdown = By.XPath("//select[@name=\"level\"]");
+
+        public readonly By language_addNew_button = By.XPath("//div[@data-tab=\"first\"]//div[@class=\"ui teal button \"]");
+        public readonly By add_button = By.XPath("//div[@data-tab=\"first\"]//input[@value=\"Add\"]");
+        public readonly By cancel_button = By.XPath("//div[@data-tab=\"first\"]//input[@value=\"Cancel\"]");
+        public By language_level_option;
+        public By lastrow_Language = By.XPath("//tr[last()]//td[1]");
+        public By lastrow_level = By.XPath("//tr[last()]//td[2]");
+
+        public By edit_button;// By.XPath("//td[text()='Mandarin']/following-sibling::td[@class='right aligned']//i[@class='outline write icon']");
+        public By delete_button;// By.XPath("//td[text()='Mandarin']/following-sibling::td[@class='right aligned']//i[@class='remove icon']");
+
+      
+
+        public By update_button = By.XPath("//div[@data-tab=\"first\"]//input[@value=\"Update\"]");
+
+        public By active_tab = By.XPath("//div[contains(@class, 'ui bottom attached tab segment') and contains(@class, 'active')]");
+
+
+        public void NavigateToLanguageTab(IWebDriver driver)
```

Comment:
Please do not pass the driver In action methods (methods which in pages doing things like create, delete etc ) instead get the pages to extend to common driver where you will create a common driver class which will initialise the web driver 

public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }

Reply:
please fix this everywhere

Reply:
Yes, All done. Thanks